From f6057596c944ec0f4ae4e7864c8ce9b91e4b37a1 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Wed, 19 Sep 2012 15:48:44 +0800
Subject: [PATCH 057/602] ENGR00224965 Update fsl git for jb bring up

Pass androidboot.hardware into boot parameter to make init
load init.freescale.rc
Disable WIFI build, which will be enabled after new wifi sdk integration
Disable the imx tinyalsa and hwc, which will be enabled after refined
Change the /dev/graphics/galcore permission, which need to be
accessed by app for hardware render

Change-Id: I13206a5e2ba80c946981e7241b55b3162eab4b71
Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 arm2_6dq/BoardConfig.mk                |  2 +-
 arm2_6sl/BoardConfig.mk                |  2 +-
 imx53_smd/BoardConfig.mk               |  2 +-
 imx5x/BoardConfigCommon.mk             |  2 +-
 imx6/BoardConfigCommon.mk              |  2 +-
 imx6/etc/ueventd.freescale.rc          |  2 +-
 imx6/sabresd/SabreSDBoardConfigComm.mk | 12 ++++++------
 imx6/soc/imx6dq.mk                     |  2 +-
 sabresd_6dq/BoardConfig.mk             |  4 ++--
 9 files changed, 15 insertions(+), 15 deletions(-)

diff --git a/arm2_6dq/BoardConfig.mk b/arm2_6dq/BoardConfig.mk
index ea05d1e..5ad7d1f 100755
--- a/arm2_6dq/BoardConfig.mk
+++ b/arm2_6dq/BoardConfig.mk
@@ -6,5 +6,5 @@ include device/fsl/imx6/soc/imx6dq.mk
 include device/fsl/imx6/arm2/BoardConfigArm2.mk
 include device/fsl-proprietary/gpu-viv/fsl-gpu.mk
 
-BOARD_KERNEL_CMDLINE := console=ttymxc0,115200 init=/init androidboot.console=ttymxc0
+BOARD_KERNEL_CMDLINE := console=ttymxc0,115200 init=/init androidboot.console=ttymxc0 androidboot.hardware=freescale
 TARGET_BOOTLOADER_CONFIG := 6q:mx6q_arm2_android_config 6dl:mx6dl_arm2_config
diff --git a/arm2_6sl/BoardConfig.mk b/arm2_6sl/BoardConfig.mk
index e2ae13b..917d069 100755
--- a/arm2_6sl/BoardConfig.mk
+++ b/arm2_6sl/BoardConfig.mk
@@ -10,5 +10,5 @@ USE_CAMERA_STUB := true
 BOARD_HAVE_IMX_CAMERA := false
 PRODUCT_MODEL := ARM2-MX6SL
 
-BOARD_KERNEL_CMDLINE := console=ttymxc0,115200 init=/init androidboot.console=ttymxc0
+BOARD_KERNEL_CMDLINE := console=ttymxc0,115200 init=/init androidboot.console=ttymxc0 androidboot.hardware=freescale
 TARGET_BOOTLOADER_CONFIG := mx6sl:mx6sl_arm2_android_config
diff --git a/imx53_smd/BoardConfig.mk b/imx53_smd/BoardConfig.mk
index ca9ea37..54a2a2e 100755
--- a/imx53_smd/BoardConfig.mk
+++ b/imx53_smd/BoardConfig.mk
@@ -57,7 +57,7 @@ TARGET_USERIMAGES_SPARSE_EXT_DISABLED := true
 BOARD_USE_AR3K_BLUETOOTH := true
 USE_OPENGL_RENDERER := true
 
-BOARD_KERNEL_CMDLINE := console=ttymxc0,115200 init=/init androidboot.console=ttymxc0 video=mxcdi1fb:RGB666,LDB-XGA ldb=di1 di1_primary pmem=128M,64M fbmem=10M gpu_memory=128M vmalloc=576M fs_sdcard=1 
+BOARD_KERNEL_CMDLINE := console=ttymxc0,115200 init=/init androidboot.console=ttymxc0 video=mxcdi1fb:RGB666,LDB-XGA ldb=di1 di1_primary pmem=128M,64M fbmem=10M gpu_memory=128M vmalloc=576M fs_sdcard=1 androidboot.hardware=freescale 
 
 BOARD_KERNEL_BASE := 0x70800000
 
diff --git a/imx5x/BoardConfigCommon.mk b/imx5x/BoardConfigCommon.mk
index a0a9a94..ce74ecd 100755
--- a/imx5x/BoardConfigCommon.mk
+++ b/imx5x/BoardConfigCommon.mk
@@ -17,7 +17,7 @@ TARGET_KERNEL_DEFCONF := imx5_android_defconfig
 TARGET_PROVIDES_INIT_RC := true
 
 #BOARD_USES_GENERIC_AUDIO := true
-BOARD_USES_ALSA_AUDIO := true
+#BOARD_USES_ALSA_AUDIO := true
 BUILD_WITH_ALSA_UTILS := true
 BOARD_HAVE_BLUETOOTH := true
 USE_CAMERA_STUB := false 
diff --git a/imx6/BoardConfigCommon.mk b/imx6/BoardConfigCommon.mk
index 29d0ff2..e9788ea 100644
--- a/imx6/BoardConfigCommon.mk
+++ b/imx6/BoardConfigCommon.mk
@@ -17,7 +17,7 @@ TARGET_PROVIDES_INIT_RC := true
 BOARD_SOC_CLASS := IMX6
 
 #BOARD_USES_GENERIC_AUDIO := true
-BOARD_USES_ALSA_AUDIO := true
+#BOARD_USES_ALSA_AUDIO := true
 BUILD_WITH_ALSA_UTILS := true
 BOARD_HAVE_BLUETOOTH := true
 USE_CAMERA_STUB := false
diff --git a/imx6/etc/ueventd.freescale.rc b/imx6/etc/ueventd.freescale.rc
index 80256da..5795f20 100644
--- a/imx6/etc/ueventd.freescale.rc
+++ b/imx6/etc/ueventd.freescale.rc
@@ -19,7 +19,7 @@
 /dev/ttya1                0660   system     gps
 /dev/ptya1                0660   system     gps
 /dev/ion                  0660   media      system
-/dev/graphics/galcore	  0660 	 system     graphics
+/dev/graphics/galcore	  0666 	 system     graphics
 /dev/graphics/fb0         0660   system     graphics
 
 # sysfs properties
diff --git a/imx6/sabresd/SabreSDBoardConfigComm.mk b/imx6/sabresd/SabreSDBoardConfigComm.mk
index b092fa9..a0d06be 100755
--- a/imx6/sabresd/SabreSDBoardConfigComm.mk
+++ b/imx6/sabresd/SabreSDBoardConfigComm.mk
@@ -9,14 +9,14 @@ TARGET_BOOTLOADER_BOARD_NAME := SABRESD
 PRODUCT_MODEL := SABRESD-MX6DQ
 
 # Wifi
-BOARD_WLAN_VENDOR 			 := ATHEROS
+#BOARD_WLAN_VENDOR 			 := ATHEROS
 # for atheros vendor
 ifeq ($(BOARD_WLAN_VENDOR),ATHEROS)
 BOARD_WLAN_DEVICE			 := ar6003
 BOARD_HAS_ATH_WLAN 			 := true
 BOARD_WLAN_ATHEROS_SDK			 := system/wlan/atheros/compat-wireless
-WPA_SUPPLICANT_VERSION			 := VER_0_9_ATHEROS
-HOSTAPD_VERSION				 := VER_0_9_ATHEROS
+WPA_SUPPLICANT_VERSION			 := VER_0_8_X
+HOSTAPD_VERSION				 := VER_0_8_ATHEROS
 WIFI_DRIVER_MODULE_PATH          	 := "/system/lib/modules/ath6kl_sdio.ko"
 WIFI_DRIVER_MODULE_NAME          	 := "ath6kl_sdio"
 WIFI_DRIVER_MODULE_ARG           	 := "suspend_mode=3 ath6kl_p2p=1"
@@ -38,9 +38,9 @@ WIFI_DRIVER_MODULE_PATH          	 := "/system/lib/modules/iwlagn.ko"
 WIFI_DRIVER_MODULE_NAME          	 := "iwlagn"
 WIFI_DRIVER_MODULE_PATH			 ?= auto
 endif
-BOARD_WPA_SUPPLICANT_DRIVER      	 := NL80211
-BOARD_HOSTAPD_DRIVER             	 := NL80211
-WIFI_TEST_INTERFACE			 := "sta"
+#BOARD_WPA_SUPPLICANT_DRIVER      	 := NL80211
+#BOARD_HOSTAPD_DRIVER             	 := NL80211
+#WIFI_TEST_INTERFACE			 := "sta"
 
 BOARD_MODEM_VENDOR := AMAZON
 
diff --git a/imx6/soc/imx6dq.mk b/imx6/soc/imx6dq.mk
index adfb801..28b32d4 100644
--- a/imx6/soc/imx6dq.mk
+++ b/imx6/soc/imx6dq.mk
@@ -11,7 +11,7 @@ BOARD_KERNEL_BASE := 0x10800000
 TARGET_KERNEL_DEFCONF := imx6_android_defconfig
 HAVE_FSL_IMX_CODEC := true
 TARGET_HAVE_IMX_GRALLOC := true
-TARGET_HAVE_IMX_HWCOMPOSER = true
+TARGET_HAVE_IMX_HWCOMPOSER = false
 USE_OPENGL_RENDERER := true
 TARGET_CPU_SMP := true
 
diff --git a/sabresd_6dq/BoardConfig.mk b/sabresd_6dq/BoardConfig.mk
index ef11ca6..266fea7 100755
--- a/sabresd_6dq/BoardConfig.mk
+++ b/sabresd_6dq/BoardConfig.mk
@@ -6,12 +6,12 @@ include device/fsl/imx6/soc/imx6dq.mk
 include device/fsl/imx6/sabresd/SabreSDBoardConfigComm.mk
 include device/fsl-proprietary/gpu-viv/fsl-gpu.mk
 
-BOARD_KERNEL_CMDLINE := console=ttymxc0,115200 init=/init video=mxcfb0 video=mxcfb1:off video=mxcfb2:off fbmem=10M vmalloc=400M androidboot.console=ttymxc0
+BOARD_KERNEL_CMDLINE := console=ttymxc0,115200 init=/init video=mxcfb0 video=mxcfb1:off video=mxcfb2:off fbmem=10M vmalloc=400M androidboot.console=ttymxc0 androidboot.hardware=freescale
 
 
 ifeq ($(TARGET_USERIMAGES_USE_UBIFS),true)
 #UBI boot command line.
-BOARD_KERNEL_CMDLINE := console=ttymxc0,115200 init=/init video=mxcfb0 video=mxcfb1:off video=mxcfb2:off fbmem=10M vmalloc=400M androidboot.console=ttymxc0  mtdparts=gpmi-nand:20m(bootloader),20m(bootimg),20m(recovery),-(root) gpmi_debug_init ubi.mtd=3
+BOARD_KERNEL_CMDLINE := console=ttymxc0,115200 init=/init video=mxcfb0 video=mxcfb1:off video=mxcfb2:off fbmem=10M vmalloc=400M androidboot.console=ttymxc0  mtdparts=gpmi-nand:20m(bootloader),20m(bootimg),20m(recovery),-(root) gpmi_debug_init ubi.mtd=3 androidboot.hardware=freescale 
 endif
 
 
-- 
1.8.0

